test/UnitTest/Microsoft.AspNet.OData.Test.Shared/ScaleDecimalTest.cs

```diff
@@ -0,0 +1,98 @@
+﻿using Microsoft.AspNet.OData.Builder;
+using Microsoft.AspNet.OData.Extensions;
+using Microsoft.AspNet.OData.Test.Abstraction;
+using Microsoft.AspNet.OData.Test.Builder.TestModels;
+using Microsoft.OData.Edm;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Xunit;
+using System.Globalization;
```

Comment:
can you add the copy right

and sorting the using

Reply:
added and sorted